src/engine/source/vdscanner/qa/test_false_negative_data/025/input_001.json

```diff
@@ -0,0 +1,70 @@
+{
+    "type": "packagelist",
+    "agent": {
+        "id": "001"
+    },
+    "packages": [
+        {
+            "architecture": "i686",
+            "checksum": "1e6ce14f97f57d1bbd46ff8e5d3e133171a1bbce",
+            "description": " ",
+            "format": "win",
+            "groups": " ",
+            "item_id": "ec465b7eb5fa011a336e95614072e4c7f1a65a53",
+            "multiarch": " ",
+            "name": "mRemoteNG",
+            "priority": " ",
+            "scan_time": "2023/08/04 19:56:11",
+            "size": 0,
+            "source": " ",
+            "vendor": "Next Generation Software",
+            "version": "1.76.20"
+        },
+        {
+            "architecture": "i686",
+            "checksum": "1e6ce14f97f57d1bbd46ff8e5d3e133171a1bbce",
+            "description": " ",
+            "format": "win",
+            "groups": " ",
+            "item_id": "ec465b7eb5fa011a336e95614072e4c7f1a65a53",
+            "multiarch": " ",
+            "name": "Bitrix24 for Windows",
+            "priority": " ",
+            "scan_time": "2023/08/04 19:56:11",
+            "size": 0,
+            "source": " ",
+            "vendor": "Bitrix",
+            "version": "15.1.21.78"
+        },
```

Comment:
@GabrielEValenzuela Please separate this into a different step or test

Reply:
Done!